README.md

```diff
@@ -37,124 +71,378 @@ To install the global command line interface (CLI), use npm global installation:
-| reportType | string | yes | Type of MeasureReport to generate: "summary", "subject-list" (not yet supported), or "individual". |
-| calculateSDEs | boolean | yes | Include Supplemental Data Elements in calculation. Defaults to true. |
-| calculateHTML | boolean | yes | Include HTML structure for highlighting. Defaults to true. |
-| calculateClauseCoverage | boolean | yes | Include HTML structure with coverage highlighting. Defaults to true. |
-| vsAPIKey | string | yes | API key, to be used to access a ValueSet API for downloading any missing ValueSets |
-| useValueSetCaching | boolean | yes | Whether to cache ValueSets obtained by an API on the filesystem. Defaults to false. |
-| useElmJsonsCaching | boolean | yes | Whether to cache the ELM JSONs and associated information from calculation. Defaults to false. |
-| clearElmJsonsCache | boolean | yes | Whether to clear the ELM JSONs cache before running calculation. Defaults to false. |
-| trustMetaProfile | boolean | yes | To "trust" the content of meta.profile as a source of truth for what profiles the data that [cql-exec-fhir](https://github.com/projecttacoma/cql-exec-fhir) grabs validates against. Defaults to false.|
-
-### CLI
-
-To run the globally installed CLI (see above), use the global `fqm-execution command`
 
-```bash
+- `[calculateClauseCoverage]`<[boolean]()>: Include HTML structure with clause coverage highlighting (default: `true`)
+- `[calculateHTML]`<[boolean]()>: Include HTML structure for highlighting (default: `true`)
+- `[calculateSDEs]`<[boolean]()>: Include Supplemental Data Elements (SDEs) in calculation (default: `true`)
+- `[clearElmJsonsCache]`<[boolean]()>: if `true`, clears ELM JSON cache before running calculation (default: `false`)
+- `[enableDebugOutput]`<[boolean]()>: Enable debug output including CQL, ELM, results (default: `false`)
+- `[includeClauseResults]` <[boolean]()>: Option to include clause results (default: `false`)
+- `[measurementPeriodEnd]`<[string]()>: End of measurement period. Defaults to the `.effectivePeriod.end` on the `Measure` resource, but can be overridden or specified using this option, which will take precedence
+- `[measurementPeriodStart]`<[string]()>: Start of measurement period. Defaults to the `.effectivePeriod.start` on the `Measure` resource, but can be overridden or specified using this option, which will take precedence
+- `[patientSource]`<[DataProvider](https://github.com/cqframework/cql-execution/blob/e4d3f24571daf3ae9f891df11bb22fc964f6de5d/src/types/cql-patient.interfaces.ts#L8)>: PatientSource to use. **If provided, the `patientBundles` argument must be `[]`**. See the [Custom PatientSource section](#custom-patientsource) for more info
+- `[reportType]`<['individual' | 'subject-list' | 'summary']()>: The type of FHIR MeasureReport to return (default: `'individual'`)
```

Comment:
Should we mention that `subject-list` is not fully supported at this time?

Reply:
What if I just..... removed it from the docs until it's supported ;)

Reply:
What a good idea... I like the way you think mgramigna 🧠